framework/base/Component.php

```diff
@@ -481,6 +483,12 @@ public function hasEventHandlers($name)
     public function on($name, $handler, $data = null, $append = true)
     {
         $this->ensureBehaviors();
+        if (is_array($handler) && !ArrayHelper::isCallable($handler)) {
+            foreach ($handler as $single) {
+                $this->on($name, $single, $data, $append);
+            }
+            return;
```

Comment:
can't this solution be done without a new call to on(), by putting a handler always inside an array?